dev_docs/tutorials/debugging.mdx

```diff
@@ -61,3 +61,49 @@ logging:
     - name: elasticsearch.query
       level: debug
 ```
+
+## Debugging Kibana with APM
+
+Kibana is integrated with APM's node and RUM agents. 
+To learn more about how APM works and what it reports, refer to the [documentation](https://www.elastic.co/observability/solution-overview).
+
+We currently track the following transactions from Kibana: 
+
+Backend (APM Node): 
+ * `request` - APM automatically tracks all incoming API requests
```

Comment:
Simplify?

```suggestion
 * `request` - tracks all incoming API requests
```

This would make the structure of list items consistent ("tracks X")